frontend/components/site/siteBuilds.jsx

```diff
@@ -89,41 +97,56 @@ function branchLink(build) {
 
 function summarizeTaskResults(build) {
   if (!build.BuildTasks || build.BuildTasks.length < 1) {
-    return (
-      <span> No scan queued </span>
-    );
+    return false;
   }
 
   const tasksWithResults = build.BuildTasks.filter(task => task.status === 'success');
+  const tasksStillProcessing = build.BuildTasks.filter(task => task.status === 'processing');
   const allTasksErrored = build.BuildTasks.every(task => task.status === 'error');
 
-  if (tasksWithResults.length > 0) {
-    const totalResults = tasksWithResults.reduce((results, task) => results + task.count, 0);
+  function totalResults(results) {
     return (
       <>
-        { resultLink(build) }
-        <span>
-          {' '}
-          (
-          { totalResults }
+        {' '}
+        <Link className="result-link" to={`/sites/${build.site.id}/builds/${build.id}/scans`}>
+          { results.reduce((acc, task) => acc + task.count, 0) }
           {' '}
-          issues)
-        </span>
+          issues found
+        </Link>
+        {' '}
       </>
     );
   }
-
-  if (allTasksErrored) {
-    return (
-      <span> Scan canceled </span>
-    );
+  let summaryVerb = 'queued';
+  if (allTasksErrored) summaryVerb = 'cancelled';
+  if (tasksStillProcessing.length > 0) summaryVerb = 'processing';
+  if (tasksWithResults.length > 0) {
+    summaryVerb = 'finished with';
   }
-
   return (
-    <span> Scan queued </span>
+    <div>
+      <b className="label-new">
+        <IconExperiment />
+        Public beta:
+      </b>
+      {' '}
+      {build.BuildTasks.length}
+      {' '}
+      build scan(s)
+      {' '}
+      {summaryVerb}
+      {tasksWithResults.length > 0 && (totalResults(tasksWithResults))}
```

Comment:
Yeah, I wondered if you'd catch that. ;)  Would we want to say that the build scans are still in progress if _any_ are in progress, instead of complete if _any_ are complete?

Reply:
A few ideas:
- If we have the horizontal screen real estate, ideal is `1 scan processing, 1 complete with X issues`
- Maybe next best is `1 complete with X issues` which doesn't tell the user that another is still processing but they'll have the link which will show them
- Any other option feels like a lie but I could be swayed